### PR TITLE
autoperibolossync: add a README

### PR DESCRIPTION
**Pull request by petr-muller**

### CODE REVIEW

File: cmd/autoperibolossync/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+`autoperibolossync` maintains the Peribolos configuration stored in a GitHub repository by executing
+the [private-org-peribolos-sync](../private-org-peribolos-sync) over a working copy of a repository containing Peribolos
+config and filing a PR to that repository if the underlying tool changes the config.
+
+## Why it exists
+
+Together with [private-org-peribolos-sync](../private-org-peribolos-sync) it
+manages [Peribolos](https://github.com/kubernetes/test-infra/tree/master/prow/cmd/peribolos) configuration of the
+GitHub repositories in
+the [openshift-priv](https://docs.ci.openshift.org/docs/architecture/private-repositories/#openshift-priv-organization).
```
Comment (smg247) [inline cmd/autoperibolossync/README.md:16]:
```suggestion
the [openshift-priv](https://docs.ci.openshift.org/docs/architecture/private-repositories/#openshift-priv-organization) organization. 
```

File: cmd/autoperibolossync/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+At the time when [private-org-peribolos-sync](../private-org-peribolos-sync) was written, there was no shared bump PR
+package available, so we tended to write `config-changing-tool | auto-config-changing-tool` pairs when we needed to
+maintain certain configuration automatically in GitHub repositories rather than include the PR filing logic in each
+configuration changing binary.
+
+## How it works
+
+`autoperibolossync` runs [private-org-peribolos-sync](../private-org-peribolos-sync) over a working copy of a repository
+containing Peribolos config. If the execution results in Peribolos changes, the tool submits or updates a pull request
+in that repository that updates the mainline with the changes, using the
```
Comment (smg247) [inline cmd/autoperibolossync/README.md:27]:
```suggestion
in that repository that updates the mainline with the changes, using
```

**Comment by petr-muller** [comment]
/label tide/merge-method-squash

Thanks!

**Pull request merged**